hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -338,10 +338,6 @@ public StateView(
 				info.setCustomFees(customFeesBuilder);
 			}
 
-			if (token.hasCustomFeeKey()) {
-				info.setCustomFeesKey(asKeyUnchecked(token.getCustomFeeKey()));
-			}
```

Comment:
I noticed that this change is applied in `master` as well. I guess that it has been decided to NOT return the fee key anymore?